kyo-offheap/native/build.sbt

```diff
@@ -0,0 +1,15 @@
+   // kyo/kyo-offheap/native/build.sbt
+   name := "KyoOffheapNative"
+
+   version := "0.1.0"
+
+   scalaVersion := "2.13.10" // Use a stable Scala version
+
+   // Enable Scala Native plugin
+   enablePlugins(ScalaNativePlugin)
+
+   // Add Scala Native dependency
+   libraryDependencies += "org.scala-native" %% "scala-native" % "0.4.13" // Use the latest stable version
```

Comment:
The latest stable is 0.5.6 https://github.com/scala-native/scala-native/releases

And I don't think you need to put it in `build.sbt`, the `sbt-scala-native` sbt plugin in `project/plugins.sbt` should be enough.

Reply:
Yes, we don't use separate sbt builds for modules. This PR shouldn't require build other than enabling `NativePlatform` for `kyo-offheap`.